iquip/apps/monitor.py

```diff
@@ -209,6 +211,146 @@ def run(self):
             logger.exception("Failed to set the level of the target TTL channel.")
 
 
+class DACControllerWidget(QWidget):
+    """Single DAC channel controller widget.
+    
+    Attributes:
+        slider: Slider for setting the voltage.
+        spinbox: Spin box for setting and showing the voltage in slider.
+        button: Button for applying the voltage in practice.
+
+    Signals:
+        voltageSet(voltage): Current voltage value is set to voltage.
+    """
+
+    voltageSet = pyqtSignal(float)
+
+    def __init__(
+        self,
+        name: str,
+        device: str,
+        channel: int,
+        ndecimals: int = 2,
+        minVoltage: float = -10,
+        maxVoltage: float = 10,
```

Comment:
Then please set the default value to 9.99.

Reply:
As I computed, the resolution of DAC is about 3mV and the available maximum voltage is about 9.9996948V.
So I set the maximum value to 9.9997 and the number of decimals to 4.

Reply:
Then setting the voltage to 9.9997 works?

Reply:
Yes! It works well and sets the actual maximum value of DAC.

Reply:
Good!